### ISSUE TITLE
Unable to Mock in Application Tests

### ISSUE THREAD

**Issue by mcordingley** [DivineOmega/laravel-password-exposed-validation-rule#17]
I'm attempting to use this library in a project. I'd like to be able to mock it during unit tests, so my local tests don't try to reach out to external systems. However, the library exposes no hooks to slip in a mock during testing. I tried to skirt the issue by resolving an instance of the rule from the Laravel container, but that ended up bypassing the internal factory method.

**Comment by DivineOmega** [comment]
Good point. Have you got any thoughts of how you'd like to see this implemented?

**Comment by mcordingley** [comment]
I think reworking it to more heavily lean on #11 would do the trick.
